server/test/services/export/CsvExporterServiceTest.java

```diff
@@ -153,6 +153,7 @@ public void programCsv_noRepeatedEntities() throws Exception {
             "applicant name (first_name)",
             "applicant name (middle_name)",
             "applicant name (last_name)",
+            "applicant name (name_suffix)",
```

Comment:
Can we keep this as `applicant name (suffix)`? You'll have to re-write it in the same place we re-write the currency from currency_cents to currency_dollars.

https://github.com/civiform/civiform/blob/d59163464e5151cd2a575a9f0cff65f17760ee29/server/app/services/export/CsvExporterService.java#L310-L312